backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/mobileui/picking/Pick_mixed_lines.feature

```diff
@@ -121,13 +121,13 @@ Feature: mobileUI Picking - Pick mixed lines
     And pick lines
       | PickingLine.byProduct | PickFromHU    | QRCode                     |
       | catchWeightProduct    | catchWeightHU | LMQ#1#0.101#08.11.2025#500 |
-      | catchWeightProduct    | catchWeightHU | LMQ#1#0.102#08.11.2025#501 |
-      | catchWeightProduct    | catchWeightHU | LMQ#1#0.103#08.11.2025#502 |
-      | catchWeightProduct    | catchWeightHU | LMQ#1#0.104#08.11.2025#503 |
-      | catchWeightProduct    | catchWeightHU | LMQ#1#0.105#08.11.2025#504 |
-      | catchWeightProduct    | catchWeightHU | LMQ#1#0.106#08.11.2025#505 |
-      | catchWeightProduct    | catchWeightHU | LMQ#1#0.107#08.11.2025#506 |
-      | catchWeightProduct    | catchWeightHU | LMQ#1#0.108#08.11.2025#507 |
+      | catchWeightProduct    | catchWeightHU | LMQ#1#0.102#08.11.2025#500 |
+      | catchWeightProduct    | catchWeightHU | LMQ#1#0.103#08.11.2025#500 |
+      | catchWeightProduct    | catchWeightHU | LMQ#1#0.104#08.11.2025#500 |
+      | catchWeightProduct    | catchWeightHU | LMQ#1#0.105#08.11.2025#500 |
+      | catchWeightProduct    | catchWeightHU | LMQ#1#0.106#08.11.2025#500 |
+      | catchWeightProduct    | catchWeightHU | LMQ#1#0.107#08.11.2025#500 |
+      | catchWeightProduct    | catchWeightHU | LMQ#1#0.108#08.11.2025#500 |
```

Comment:
why did u change this test and instead of having different lot numbers now we have the same lot no?

Reply:
if we have different lot numbers you end up with one shipment line for each lot number, now I'm not sure if that's the goal of this test, I can also update the test to leave the lot numbers in there and update the shipment line validation to match what actually happens, i.e. instread of having 2 lines, we will get one for each diff lot number

Reply:
@cp-ps having different lot numbers in scanned L+M barcodes is a common case.
What to do with that, depends on your requirements, 
but for sure changing the test input to match your expected output is a no go.

Reply:
>  your expected output is a no go.

it's not my expected output, it was the test expected output, anyway, I'll go with changing the output then, i.e. creating multiple shipment lines.

Reply:
@cp-ps  if you keep this cucmber test as it is, it's fine, but it won't test your case.
So either you come up with a new test to cover you case, 
or (loud thinking) I would change/add some of the L+M barcodes to have the same LotNo and ofc expect those with the same LotNo to be grouped.

anyways, feel free to come up with the right tests to cover your cases of this issue.

Reply:
more, do we have to consider de.metas.inoutcandidate.model.I_M_ShipmentSchedule_AttributeConfig for these tests too?

Reply:
> anyways, feel free to come up with the right tests to cover your cases of this issue.

I did, it's in the new cucumber I've added: `backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/mobileui/picking/Pick_with_Attributes.feature`

Reply:
> more, do we have to consider de.metas.inoutcandidate.model.I_M_ShipmentSchedule_AttributeConfig for these tests too?

yes, very good point, right now LotNo is part of that in the vanila DB, but I agree, it's not something written in stone, I'll add a step to set that